install/README.md

```diff
@@ -85,6 +85,7 @@ Manual installation instructions are available for [these distributions](#suppor
 ## Installing on a Cloud Service Platform
 * [Amazon Web Services (AWS)](aws-installation-guide.md)
 * [Google Compute Engine (GCE)](gce-installation-guide.md)
+# [VEXXHOST OpenStack Cloud](vexxhost-installation-guide.md)
```

Comment:
Hey folks - I think this title had the wrong indent level - it renders as a **TOP LEVEL ITEM**.... I'll go PR a fix...